inabox/tests/setup_disperser_harness.go

```diff
@@ -298,160 +280,60 @@ func startRelayWithListener(
 		},
 	}
 
-	relayLogger, err := common.NewLogger(&loggerConfig)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create relay logger: %w", err)
-	}
-
 	// Create AWS clients using LocalStack container's configuration
 	awsConfig := harness.LocalStack.GetAWSClientConfig()
-	dynamoClient, err := dynamodb.NewClient(awsConfig, relayLogger)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create dynamodb client: %w", err)
-	}
 
-	s3Client, err := s3.NewClient(ctx, awsConfig, relayLogger)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create s3 client: %w", err)
+	// Create eth client config with RPC URL from deployer
+	ethClientConfig := geth.EthClientConfig{
+		RPCURLs:    []string{config.TestConfig.Deployers[0].RPC},
```

Comment:
why is this under Deployers? Feel like the RPCURLs should live much higher up in the config?

Reply:
https://github.com/Layr-Labs/eigenda/pull/2107/commits/c82c18c7f995840f43ebb5845601d87c83c0de00

I pass the chain harness ethclient now